locales/base.json

```diff
@@ -37,6 +37,7 @@
     "resource": "Get rewards by supporting small businesses",
     "ethichub": "Lend capital to unbanked farmers through DeFi crowdfunding",
     "uniswap": "Swap and get rewards by providing liquidity",
+    "double": "Get rewards and earn yields from unused tokens by providing liquidity",
```

Comment:
Legal does not allow us to talk about earning yield, can we keep this as "get rewards by providing liquidity"? (wasn't sure what "unused tokens" meant so I omitted it)

Reply:
Sounds good.

Reply:
@h3he you'll need to make this change as I don't have write access to your fork.

Reply:
OK. Just made the change. Please review and confirm.